### PR TITLE
MIMIR-17 added change date and description

### PR DESCRIPTION
**Pull request by erikgus**

### CODE REVIEW

File: src/main/resources/site/parts/statistics/statistics.es6 [hunk 1]
```diff
@@ -59,21 +64,45 @@ const renderPart = (req) => {
...
   }
 
-  const body = render(view, model)
-  return {
-    body,
+  let body = render(view, model)
+  body = modifiedDateComponent.renderBody({
+    body
+  })
+  pageContributions = modifiedDateComponent.renderPageContributions({
```
Comment (johnnadeluy) [inline src/main/resources/site/parts/statistics/statistics.es6:97]: Lint: _'error  'pageContributions' is never reassigned. Use 'const' instead  prefer-cons'_

You can just delete 'let pageContributions' and use const pageContributions here

**Review by johnnadeluy — CHANGES_REQUESTED** [review]
Aside from the tiny lint nitpick, the placement of modified date is a bit off i.e. modified date should be on the same line as 'updated' date not under 'next update' date on desktop, according to the zeplin sketches. I'll further comment on this in the JIRA-task.

File: src/main/resources/site/parts/statistics/statistics.es6 (hunk 1 above)
Comment (erikgus) [reply src/main/resources/site/parts/statistics/statistics.es6]: fixed

**Comment by erikgus** [comment]
Fixed position of element and fixed lint error

**Review by johnnadeluy — APPROVED** [review]
Looks good! Good job 🙌

**Pull request merged**